README.md

```diff
@@ -0,0 +1,108 @@
+# Expresso Weather
+
+Expresso Weather is a JSON API that uses the Dark Sky API along with the Google Geocoding API to return weather information based on a location provided by the user. The user must include a valid API key. The data returned includes houlry and daily objects base on the current weather. These objects include a summary, icon, temperature, percipitation percentages, as well as wind and visibility information. 
```

Comment:
You need to change houlry to hourly!

Reply:
I like how you described the objects information.

Reply:
Thanks!